modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -0,0 +1,304 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLink from '@clayui/link';
+import ClaySticker from '@clayui/sticker';
+import ClayTabs from '@clayui/tabs';
+import React, {useMemo, useState} from 'react';
+
+import Sidebar from './Sidebar';
+
+const formatDate = (date, languageTag) => {
+	return (
+		date &&
+		languageTag &&
+		Intl.DateTimeFormat(languageTag, {
+			day: 'numeric',
+			hour: 'numeric',
+			hour12: true,
+			minute: 'numeric',
+			month: 'short',
+			year: 'numeric',
+		}).format(new Date(date))
+	);
+};
+
+const SidebarPanelInfoView = ({
+	categories = [],
+	classPK,
+	createDate,
+	data = {},
+	languageTag = 'en',
+	modifiedDate,
+	subType,
+	tags = [],
+	title,
+	userName,
+	userPortraitURL,
+	versions = [],
+	viewURLs = [],
+}) => {
+	const [activeTabKeyValue, setActiveTabKeyValue] = useState(0);
+
+	const formattedCreateDate = useMemo(() => {
+		return formatDate(createDate, languageTag);
+	}, [createDate, languageTag]);
+
+	const formattedDisplayDate = useMemo(() => {
+		return formatDate(data['display-date']?.value, languageTag);
+	}, [data, languageTag]);
+
+	const formattedExpirationDate = useMemo(() => {
+		return formatDate(data['expiration-date']?.value, languageTag);
+	}, [data, languageTag]);
+
+	const formattedModifiedDate = useMemo(() => {
+		return formatDate(modifiedDate, languageTag);
+	}, [modifiedDate, languageTag]);
+
+	const formattedReviewDate = useMemo(() => {
+		return formatDate(data['review-date']?.value, languageTag);
+	}, [data, languageTag]);
+
+	const sortedViewURLS = useMemo(() => {
+		return viewURLs
+			.sort((a, b) => {
+				if (a.languageId < b.languageId) {
```

Comment:
Not sure, but chances are `languageId` is a string, right? If not, this could be simplified to

```js
return a.languageId - b.languageId;
```

Reply:
yep, they're strings